win32/src/file_win32.c

```diff
@@ -60,4 +41,472 @@ static void on_file_io_complete_win32(
     ULONG io_result,
     ULONG_PTR number_of_bytes_transferred,
     PTP_IO io
-);
\ No newline at end of file
+);
+
+static VOID NTAPI on_close_threadpool_group_member(
+     PVOID object_context, /*what was passed @  CreateThreadpoolIo*/

```

Comment:
>      [](start = 0, length = 5)

nit: extra spaces detected (5 instead of 4)

Reply:
done

---
In reply to: [456704121](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456704121) [](ancestors = 456704121)